config.md

```diff
@@ -143,3 +143,26 @@ clang's parser, such as incoming references.
 Whether files are built in the background to produce a project index.
 This is checked for translation units only, not headers they include.
 Legal values are `Build` (the default) or `Skip`.
+
+### External
+
+Used to define an external index source. This can be:
+- An on-disk monolithic index produced by `clangd-indexer` or
+- Address of a remote-index-server.
+`MountPoint` can be used to specify source root for the index. This is necessary
+to handle relative path conversions. Overall schema looks like this:
+
+```
+Index:
+  External:
+    File: /abs/path/to/an/index.idx
```

Comment:
```
File: /abs/path/to/an/index.idx
# OR
Server: my.index.server.com:50051
```

since you have `File` or `Server` below, otherwise it's slightly confusing